api/Navigator.json

```diff
@@ -3742,16 +3742,28 @@
             },
             "chrome_android": "mirror",
             "edge": "mirror",
-            "firefox": {
-              "version_added": "97",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "dom.webmidi.enabled",
-                  "value_to_set": "true"
-                }
-              ]
-            },
+            "firefox": [
+              {
+                "version_added": "108",
+                "notes": "API access is gated by installation of a <a href='https://support.mozilla.org/en-US/kb/site-permission-addons'>site permission add-on</a> (user prompt), secure context, and <a href='https://developer.mozilla.org/docs/Web/HTTP/Headers/Feature-Policy/midi'>Permission Policy: <code>midi</code></a>."
```

Comment:
FYI, this merged in from changes in https://github.com/mdn/browser-compat-data/pull/18160 (now closed)